libconcorde_tsp_solver/common/files/TSPlib_file.txt

```diff
@@ -0,0 +1,11 @@
+NAME: ipa-building-navigation
```

Comment:
This was a necessary auto generated file, but we need write access - so now we create this file in ~/.ros.